jupytercalpoly/Jupyter-Bifrost
# Customize tab scroll

Fix #43 , #32 , #50 

## Changes
* Fixed size of the side bar getting bigger as the sidebar content gets bigger.
* Fixed scrollintoview not working in chrome
    * Recent version of chrome doesn't support smooth behavior per [this](https://stackoverflow.com/questions/61885401/scrollintoview-is-not-working-in-chrome-version-81-behaviour-smooth-is-not-hap)
* Fixed vertical scroll of customize mark screen.

## ScreenShot
![Kapture 2021-07-20 at 10 45 31](https://user-images.githubusercontent.com/15991814/126372463-7a5f421f-f363-40d2-bf15-1bfb67b2c8f6.gif)


## Testing
1. plot bifrost()
2. Select columns
3. Scroll through the recommended charts through arrow keys
4. Hit customize tab
5. See changes
